jetty-core/jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -693,7 +693,12 @@ else if (properties.size() > 0)
                 // TODO module path
 
                 for (Prop property : environment.getProperties())
-                    cmd.addArg(property.key + "=" + property.value);
+                {
+                    String value = property.value;
+                    if (isDryRun() && value != null && value.contains("$"))
+                        value = "'" + value + "'";
+                    cmd.addArg(property.key + "=" + value);
```

Comment:
This feels a bit arbitrary.  What if the value contains space?  what if the value contains `'` or `"`?    Isn't there a method already that checks if a string needs to be quoted and then quotes it correctly, perhaps escaping quote characters?